### ISSUE TITLE
Issue with "ElementInternals": ElementInternals.role is missing

### ISSUE THREAD

**Issue by kantoche** [mdn/content#13355]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals

The property **ElementInternals.role** is missing in the section [Properties included from ARIA](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals#properties_included_from_aria)
and in the table of [Browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals#browser_compatibility)

Related: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/elementinternals`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/elementinternals/index.md
* Last commit: https://github.com/mdn/content/commit/7aad7d9db1bff042e80985bd7e6bb54d96c8cb93
* Document last modified: 2022-02-27T22:06:00.000Z

</details>

**Comment by ordehi** [comment]
Please assign this to me, I'd like to do it.

**Comment by Josh-Cena** [comment]
Fixed by https://github.com/mdn/content/pull/20182 and https://github.com/mdn/browser-compat-data/pull/17498
